pages/osx/brew.md

```diff
@@ -14,14 +14,18 @@
 
 `brew list`
 
-- Update an installed formula (passing no formula updates all installed formulae):
+- Update an installed formula (passing no `{{formula}}` updates all installed formulae):
 
 `brew upgrade {{formula}}`
 
-- Update brew itself:
+- Fetch the newest version of Homebrew and all formulae from GitHub:
```

Comment:
No change in name. Just using the full name `Homebrew` instead of just `brew`. Again, the full name is already being using in the example/descriptions in the current/active version of `pages/osx/brew.md`